apps/devhub/tests/test_views.py

```diff
@@ -2539,9 +2539,9 @@ def xpi(self):
                     'rb')
 
     @mock.patch.object(waffle, 'flag_is_active')
+    @mock.patch('devhub.tasks.validate')
```

Comment:
Oh, er, and I guess I should have re-read the function before writing those comments, since it doesn't actually test polling before the task completes. :( So I guess I'll add that too.